testing/security/transport_encryption.py

```diff
@@ -95,7 +102,7 @@ def fetch_dcos_ca_bundle_contents() -> str:
         log.error("Error fetching DC/OS CA bundle")
         raise Exception("Errot fetching DC/OS CA bundle")
 
-    return cert
+    return cert.decode('utf-8')
```

Comment:
This seems to change the semantics of the function. Should we fix the type annotation instead?

Reply:
Addressed - not sure what this was about!